matlab/library/WBToolboxLibrary_repository.mdl

```diff
@@ -1,14 +1,14 @@
 Library {
-  Name			  "WBToolboxLibrary_repository2"
+  Name			  "WBToolboxLibrary_repository"
```

Comment:
@VenusPasandi did you renamed explicitly the library? Is that intentional?

Reply:
I have no memory of this but the 2 could have been there for showing simultaneously the mdl (developement version with newer matlab version) and slx libraries (deployed version, compatible with old matlab version).

In view of this, please double check that the committed simulink libraries are exported with the correct simulink versions (that I don't remember where we specified it).

Reply:
I have no idea about this extra "2". I didn't change the name of the library.

As I checked, previously it was "WBToolboxLibrary_repository", but then it changes to "WBToolboxLibrary_repository2" in the commit a1b81001950dd1768b54d8cc9b4b1659909aea25.

>> In view of this, please double check that the committed simulink libraries are exported with the correct simulink versions (that I don't remember where we specified it).

Yes, it is correct. The exported library is in R2016b.

Reply:
Then ok, I guess that removing the 2 is ok, no idea why in that PR it was changed